src/test/java/com/example/tradingapp/TradingAppApplicationTests.java

```diff
@@ -0,0 +1,84 @@
+package com.example.tradingapp;
+
+import com.example.tradingapp.secrets.Secrets;
+import com.example.tradingapp.trading.OkxOrderRequestDecoder;
+import com.example.tradingapp.trading.OkxOrderRequestEncoder;
+import com.example.tradingapp.trading.model.OrderRequest;
+import com.example.tradingapp.trading.model.OrderType;
+import com.example.tradingapp.trading.model.Side;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.util.EntityUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+@SpringBootTest
+public class TradingAppApplicationTests {
+
+    @Autowired
+    OkxOrderRequestEncoder encoder;
+    @Autowired
+    OkxOrderRequestDecoder decoder;
+
+    @Test
+    void shouldAddHeaders() throws IOException {
+        var orderReq = OrderRequest.builder()
+                .symbol("Btc")
+                .side(Side.BUY)
+                .type(OrderType.LIMIT)
+                .price(100)
+                .quantity(20)
+                .build();
+
+        HttpPost encodedReq = (HttpPost) encoder.encode(orderReq);
+
+        Assertions.assertThat(Arrays.toString(encodedReq.getHeaders("OK-ACCESS-KEY")))
```

Comment:
consider using static import for `Assertions.assertThat`
This way instead of writing `Assertions.assertThat(...).` you just need `assertThat(...).`

Usage: Alt+Enter
![image](https://user-images.githubusercontent.com/7275902/178932832-8e188df2-961f-419e-b599-8c8b3798249a.png)